docs/source/conf.py

```diff
@@ -12,54 +12,67 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../..'))
+
+sys.path.insert(0, os.path.abspath("../.."))
 
 
 # -- Project information -----------------------------------------------------
 
-project = 'MovingPandas'
-copyright = '2023, MovingPandas developers'
-author = 'MovingPandas developers'
+project = "MovingPandas"
+copyright = "2023, MovingPandas developers"
+author = "MovingPandas developers"
 
 # The full version, including alpha/beta/rc tags
-release = 'master'
+release = "master"
```

Comment:
does this have to be main instead of master?

Reply:
No, I guess this just hasn't been updated yet

Reply:
I don't think the docs build is failing. It probably has to be changed on the readthedocs side first. I'll leave it for another PR.

Reply:
Yeah, the docs are fine. There are just a few strings that read master instead of main